pkg/operator/ceph/file/filesystem.go

```diff
@@ -180,6 +179,9 @@ func (f *Filesystem) doFilesystemCreate(context *clusterd.Context, cephVersion c
 					fmt.Sprintf(": %+v", err),
 			)
 		}
+		if err := client.SetPoolSize(context, clusterName, f.Name, f.dataPools, f.metadataPool); err != nil {
```

Comment:
no need of semicolon here.

Reply:
okay I think I misread it because of the split view of github. Seems okay to me.

Reply:
I think it is :)